.stylelintrc.yml

```diff
@@ -3,9 +3,11 @@ extends:
 plugins:
   - stylelint-order
   - stylelint-selector-pseudo-class-lvhfa
+ignoreFiles:
+  - './dist/**'
 rules:
   at-rule-no-unknown: null
-  alpha-value-notation: number
+  alpha-value-notation: percentage
```

Comment:
I changed it to number because it was asking me to to set opacity to`0%` instead of `0` and another value that needed to be in number.  No need to complicate things.  Using 0 and 1 for opacity has always worked.

Reply:
Your call though.

Reply:
I am fine if we want to use number, I just felt like percentage was more readable.

Reply:
Are there other properties where percentage will be used?  The opacity 0% doesn't seem right to me but if there are other properties where percentages makes more sense I'm okay with it.